### ISSUE TITLE
Accessibility: aria-labels on icon buttons / links with no text content

### ISSUE THREAD

**Issue by Lane** [EvictionLab/eviction-lab-website#177]
Primary for social media links, but anything that uses only an icon should have an `aria-label` attribute on it  to indicate what the button or link is for screen readers.
